package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-ui-kit",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-ui-kit/issues",
-  "version": "4.5.0",
+  "version": "4.6.0-beta.0",
```

Comment:
see how preview looks
 - good? merge and publish 4.6.0
 - bad? ask david to update the icon so it looks good, repeat from the beginning